hook_test.go

```diff
@@ -68,3 +68,18 @@ func TestHookAddStacks(t *testing.T) {
 	logger.Debug("No stacks.")
 	assert.NotContains(t, buf.String(), "Unexpected stacktrace at Debug level.")
 }
+
+func TestHooksNilEntry(t *testing.T) {
+	tests := []struct {
+		hook hook
+		name string
+	}{
+		{hook: AddStacks(InfoLevel).(hook), name: "AddStacks"},
```

Comment:
nit: suggest putting name first, and then hook (in the struct, and inline here).

since it's also pretty simple, I'd be OK with dropping the field names in the inline struct, this seems pretty readable:
```go
{"AddStacks", AddStacks(InfoLevel).(hook)}
{"AddCaller", AddCaller().(hook)}
```